certbot/compat/filesystem.py

```diff
@@ -173,3 +241,14 @@ def _compare_dacls(dacl1, dacl2):
     """
     return ([dacl1.GetAce(index) for index in range(0, dacl1.GetAceCount())] ==
             [dacl2.GetAce(index) for index in range(0, dacl2.GetAceCount())])
+
+
+def _get_current_user():
+    """
+    Return the pySID corresponding to the current user.
+    """
+    account_name = win32api.GetUserNameEx(win32api.NameSamCompatible)
+    # Passing None to systemName instruct the lookup to start from the local system,
```

Comment:
just a note that we'll want to keep the version in 3c rather than this version

Reply:
Even if GIT would shout at us a merge conflict reminding us to synchronize the two implementations, I took the version in part3c to put it here and so deal with it right now.